packages/windbg.vm/tools/chocolateyinstall.ps1

```diff
@@ -5,17 +5,32 @@ try {
     $toolName = 'WinDbg'
     $category = 'Debuggers'
 
-    # It seems WinDbg is now distributed as an .appinstaller and we need to install it using Add-AppxPackage
-    Add-AppxPackage -AppInstallerFile 'https://windbg.download.prss.microsoft.com/dbazure/prod/1-0-0/windbg.appinstaller'
+    $bundleUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/1-2402-24001-0/windbg.msixbundle"
+    $bundleSha256 = "e941076cb4d7912d32a22ea87ad2693c01fa465227b4d1ead588283518de428f"
 
-    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
-    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-    $executableCmd  = Join-Path ${Env:WinDir} "system32\cmd.exe"
-    # Use `start` to close the open console
-    $executableArgs = "/C start WinDbgX.exe"
-    $executableDir  = Join-Path ${Env:UserProfile} "Desktop"
-    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executableCmd -Arguments $executableArgs -WorkingDirectory $executableDir -RunAsAdmin
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        url           = $bundleUrl
+        checksum      = $bundleSha256
+        checksumType  = "sha256"
+        fileFullPath  = Join-Path ${Env:TEMP} "$toolName.msixbundle"
+    }
+    Get-ChocolateyWebFile @packageArgs
+    Add-AppxPackage -Path $packageArgs.fileFullPath
+
+    $installDir = (Get-AppxPackage -Name "Microsoft.$toolName").InstallLocation
+    $iconLocation  = Join-Path $installDir "DbgX.Shell.exe" -Resolve
+    $executablePath = "$(where.exe WinDbgXA.exe)"
+    VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -iconLocation $iconLocation -RunAsAdmin
 } catch {
-    VM-Write-Log-Exception $_
+    if ($_.Exception.Message -match "INFO: Could not find files for the given pattern\(s\).")
+    {
+        $executablePath = Join-Path $installDir "DbgX.Shell.exe"
+        VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -iconLocation $iconLocation -RunAsAdmin
+    }
+    else
+    {
+        VM-Write-Log-Exception $_
+    }
```

Comment:
I'm not sure if this is the route we want to take, as noted in my comments on the PR.

I think our current route of just making a shortcut may be the better alternative for the moment, unless we want to take the time to properly test and confirm one of the possible working solutions I noted in my comments.

Reply:
At least we need to continue on this error, or `windows-2019` will fail on `where.exe WinDbgX.exe`.